server/models/poi_version.js

```diff
@@ -2,6 +2,7 @@ const CONFIG = require('config')
 const __ = CONFIG.universalPath
 const _ = __.require('lib', 'utils')
 const error_ = __.require('lib', 'error')
+const gj_tools_ = require('geojson-tools')
```

Comment:
current style guide uses camelCased variables so `gj_tools_` should probably be written `gjTools` (or just `geojson`?)